package.json

```diff
@@ -27,7 +27,7 @@
     "test:watch": "npm run test -- --watchAll"
   },
   "dependencies": {
-    "@storybook/theming": "^5.0.6",
+    "@storybook/theming": "^5.1.9",
```

Comment:
maybe we want to have this as dev dependency, I dont see any reason that this is up here.

Reply:
👍